### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bianchini88**
Updating description for "From data to publication" workshop

### CODE REVIEW

**Comment by amorris28** [comment]
Just wanted to check since you submitted this as a "draft" PR. Are you ready to merge your changes or are you still working on them? When you are ready, please mark this as "Ready for review." Thanks!

**Comment by bianchini88** [comment]
@amorris28 Thanks for checking. I am waiting for approval from the other instructor... we will have a meeting a bit later today and I will ask him.

**Review by amorris28 — APPROVED** [review]

**Pull request merged**
